app/views/links/edit.html.erb

```diff
@@ -1,44 +1,49 @@
-<% breadcrumb :links, @local_authority, @service, @interaction %>
+<% content_for :page_title, "Editing #{ @local_authority.name } - #{ @service.label}" %>
 
-<div class="page-title">
-  <h1><%= @local_authority.name %></h1>
+<%= render "govuk_publishing_components/components/heading", {
+  text: "Editing #{@local_authority.name} - #{ @service.label}",
+  heading_level: 1,
+  font_size: 'l',
+} %>
 
-  <h2><%= @service.label %></h2>
-  <p><b>Service code</b> <%= @service.lgsl_code %></p>
-</div>
-
-<% if flash[:danger] %>
-<div class="alert alert-danger"><%= flash[:danger] %></div>
-<% end %>
-
-<h3><%= @interaction.label %></h3>
+<%= render "govuk_publishing_components/components/heading", {
+  text:  @interaction.label,
+  heading_level: 2,
+  font_size: "s",
+} %>
 
 <%= render partial: "link_status_alerts", locals: { alerts: @link.link_warnings, html_class: "warning", prefix: "Warning" } %>
 <%= render partial: "link_status_alerts", locals: { alerts: @link.link_errors, html_class: "danger", prefix: "Error" } %>
 
```

Comment:
I think the alerts partial uses bootstrap `alert alert-`. It looks like the notification banner is the intended replacement for alerts https://design-system.service.gov.uk/components/notification-banner/ and https://components.publishing.service.gov.uk/component-guide/notice.